.github/workflows/lint.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           path: |
             node_modules
             packages/*/node_modules
+            metapackages/*/node_modules
```

Comment:
This seemed like the only place that likely needing something added. Nothing in the CircleCI setup seemed like it would need tweaking but perhaps I missed something.

Reply:
You also need to add metapackages to the lerna.json

Reply:
Sorry, I meant with regards to the CI configuration files (circle VS actions). I did add to lerna.json with this PR but definitely double-check I did correctly.

Reply:
Ah sorry I just saw "This seemed like the only place that likely needing something added" and my brain was on autopilot.

Reply:
No problem. I definitely should have been clearer.